modules/nixos/common/default.nix

```diff
@@ -35,6 +35,8 @@
   zramSwap.enable = true;
   zramSwap.memoryPercent = 100;
 
+  systemd.tmpfiles.rules = [ "D! /nix/var/nix/builds 0755 root root" ];
```

Comment:
You might want to only do this for old files. Otherwise running a NixOS activation while having a build running produces random crashes.

Reply:
This runs on boot, not activation?